neusomatic/python/long_read_indelrealign.py

```diff
@@ -1196,18 +1230,36 @@ def long_read_indelrealign(work, input_bam, output_bam, output_vcf, region_bed_f
         region_bed_file, extended_region_bed_file, ref_fasta_file, chrom_lengths, pad)
     region_bed_file = extended_region_bed_file
 
-    region_bed = pybedtools.BedTool(region_bed_file)
-
-    region_bed_merged = pybedtools.BedTool(region_bed)
+    region_bed_merged = region_bed_file
+    len_merged = 0
+    with open(region_bed_merged) as r_b:
+        for line in r_b:
```

Comment:
As commented earlier, a generator to skip empty lines and header lines will simplify a lot of these code.